star-rx72n-firmware/lib/rx_frame/src/rx_frame.c

```
@@ -79,6 +77,71 @@ static uint32_t internal_read_le32(const uint8_t* buf)
          ((uint32_t)buf[k_le32_byte_3] << k_shift_byte_3);
 }
 
+static rx_err_t internal_decode_header(const uint8_t* data,
+                                       const uint32_t data_len,
+                                       rx_frame_t*    frame,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_decode_header' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)